examples/rust-simple/README.md

```diff
@@ -8,62 +8,70 @@ It will
 - Every 10 seconds, write a log saying "Hi! I'm a Game Server"
 - After 60 seconds, call `SDK::Shutdown()` to shut the server down.
 
-## Running locally (without Docker)
+## Running in a cluster
 
-This will build example server which will run for 100 seconds:
-```
-make build
-```
+These instructions allow you to run the example code without building a container, but they require you to
+have a cluster with Agones installed. If you don't yet have a cluster, check out
+[these instructions](https://agones.dev/site/docs/installation/).
 
-In a separate terminal run next command:
-```
-cd ../../build; make run-sdk-conformance-local TIMEOUT=120 TESTS=ready,watch,health,gameserver
-```
-This will start an SDK-server in a docker, which will be running for 120 seconds.
 
-Run the Rust Simple Gameserver binary:
+To run the example server:
 ```
-make run
+kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/rust-simple/gameserver.yaml
```

Comment:
Same as the C++ one, should we make this 
```suggestion
kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/<release-branch>/examples/rust-simple/gameserver.yaml
```

With a note saying "Where <release-branch> should match the Agones version you have deployed, e.g `release-0.12`"

Sound good?